### ISSUE TITLE
Variable support

### ISSUE THREAD

**Issue by jmbarrios** [ropensci/ghql#18]
GraphQL support variable definition as part of a query. It is consider a best practice to be explicit about what part of a query is dynamic, as we can see in https://graphql.org/learn/queries/#variables.
Issue #17 can use this characteristic as follows:
```
qry$query('getGeneInfo', '
  query getGeneInfo($genId: String!) {
    geneInfo(geneId: $genId) {
    id
    symbol
    chromosome
    start
    end
    bioType
    __typename
  }
}')
```
and variable data
```
{
  "genId": "ENSG00000137033"
}
```
Variable data can be build as a json encode of a named list object and added to the exec query method.

**Comment by jmbarrios** [comment]
I will work on it

**Comment by sckott** [comment]
thanks @jmbarrios - sounds good, thanks very much

**Comment by sckott** [comment]
fixed in #19
